client/src/questions-n-answers/qNa.jsx

```diff
@@ -81,13 +98,29 @@ class QuestionsNAnswers extends React.Component {
       let sortedData = this.filterAnswersNQuestions(copy)
       let showButton = this.helper().showMoreAnsweredQuestions(sortedData)
 
-      this.setState({
-        questions: sortedData[0],
-        answers:sortedData[1],
-        showQuestionButton: showButton,
+      let answerIds = this.state.reported
+
 
+       sortedData[1].forEach((answer) => {
```

Comment:
You noted in comments your are not too DRY, and eventually this could be a good place for a helper func that does this work